nucypher/characters/lawful.py

```diff
@@ -119,51 +151,52 @@ def generate_kfrags(self, bob: 'Bob', label: bytes, m: int, n: int) -> List:
         :param n: Total number of kfrags to generate
         """
 
-        self.revocation_kits = dict()
-        bob_pubkey_enc = bob.public_keys(DecryptingPower)
+        params = self.generate_m_of_n(m=m, n=n)
```

Comment:
Hmm... I think, methods for such small things are only confusing the reader who doesn't like jumping over the code. Why not simply:
```
m = m or self.m
n = n or self.n
...
return delegating_power.generate_kfrags(..., m=m, n=n)
```
More readable, no? Or do we want to have more involved logic for selecting m and n eventually?